src/tests/ov_helpers/ov_snippets_models/include/subgraph_mlp.hpp

```diff
@@ -0,0 +1,73 @@
+// Copyright (C) 2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include "snippets_helpers.hpp"
+
+/* The file contains graphs with different MLP-patterns:
+ * Skeleton of MLP-pattern is:
+ *       Input
+ *         |
+ *      MatMul
+ *         |
+ *      Eltwise
+ *         |
+ *      MatMul
+ *         |
+ *      Output
+ */
+
+namespace ov::test::snippets {
+
+/* Graph:
+ *       Input
+ *         |
+ *        FQ
+ *         |
+ *      MatMul
+ *         |
+ *      Eltwise
+ *         |
+ *        ...
+ *         |
+ *      Eltwise
+ *         |
+ *        FQ
+ *         |
+ *      MatMul
+ *         |
+ *      Output
+ */
+class MLPSeqTypeRelaxedFunction : public SnippetsFunctionBase {
```

Comment:
Can we rename it to `MLPSeqQuantizedTypeRelaxedFunction`? And `MLPSeqFunction` to `MLPSeqQuantizedFunction`?
Because these models contain FakeQiantized models.

As far as I remember, we discussed about `MLPSeq` without FQ. I think we need to add such tests. I believe it won't be complicated :) Bu we extend out test coverage

Reply:
Renamed the tests, there are actually tow kinds of tests: normal and quantized type relaxed ones